### ISSUE TITLE
Is this project still maintained ?

### ISSUE THREAD

**Issue by mjrider** [Moxio/php-codesniffer-sniffs#22]
Hello,

is this project still maintained, 
i see that the last release is more then ~~a year ago~~  (sorry, the github releases didn't match the tags) 6 months , and slevomat/coding-standard has had some new versions since then.

regards
Robbert

**Comment by aboks** [comment]
Yes, it is still maintained, just maybe not that actively. We use these sniffs ourselves, just not in combination with actual sniffs from `slevomat/coding-standard`. That's why we're not really active with upgrading this dependency ourselves. The `slevomat/coding-standard` dependency is limited due to reasons outlined in https://github.com/Moxio/php-codesniffer-sniffs/pull/10#issuecomment-822543540, but we welcome PR's for raising this limit if it is verified that our sniffs still work with those newer versions.

**Comment by Potherca** [comment]
See #25
